tests/sentry/tsdb/test_snuba.py

```diff
@@ -0,0 +1,82 @@
+from __future__ import absolute_import
+
+import pytz
+from datetime import datetime, timedelta
+
+from sentry.testutils.cases import OutcomesSnubaTest
+from sentry.tsdb.base import TSDBModel
+from sentry.tsdb.snuba import SnubaTSDB
+from sentry.utils.outcomes import Outcome
+
+
+def to_epoch_time(value):
```

Comment:
We have a similar timestamp generation logic in `sentry.utils.dates.to_timestamp()` You could reuse that here.